dbms/src/Flash/Mpp/MinTSOScheduler.cpp

```diff
@@ -193,7 +199,9 @@ bool MinTSOScheduler::scheduleImp(const UInt64 tso, const MPPQueryTaskSetPtr & q
     }
     else
     {
-        if (tso <= min_tso) /// the min_tso query should fully run, otherwise throw errors here.
+        bool is_tso_min = tso <= min_tso;
+        fiu_do_on(FailPoints::random_min_tso_scheduler_failpoint, is_tso_min = true;);
```

Comment:
This failpoint looks unreasonable?

Reply:
This failpoint want to reproduce the situation that it's min_tso, while thread hard limit reached.